x/evm/keeper/statedb.go

```diff
@@ -591,7 +591,7 @@ func (k *Keeper) AddLog(log *ethtypes.Log) {
 	log.BlockHash = common.BytesToHash(k.ctx.HeaderHash())
 	log.TxIndex = uint(k.GetTxIndexTransient())
 
-	logs := k.GetTxLogs(log.TxHash)
+	logs := k.GetTxLogs(key)
```

Comment:
why this change?

Reply:
Because the `log.TxHash` is eth hash here, while `key` is the cosmos tx hash.